spec/features/results_download_spec.rb

```diff
@@ -12,8 +17,6 @@
   it "visits results page, clicks 'Waitlist', checks if csv file is downloaded" do
```

Comment:
It doesn't, it only visits results page and checks if there is a link with that text. ;)